pkgs/tools/security/gnupg/24.nix

```diff
@@ -103,6 +104,45 @@ stdenv.mkDerivation rec {
     # we can include just the configure changes relevant to the static patch
     # in the patch file.
     ./static.patch
+
+    # Fix for memory leak that is expected to be included in 2.4.9
+    (fetchpatch {
+      url = "https://git.gnupg.org/cgi-bin/gitweb.cgi?p=gnupg.git;a=patch;h=ff30683418695f5d2cc9e6cf8c9418e09378ebe4";
+      hash = "sha256-6+9QtnGufHICS0OOojbJvCdeP5NxC6wjlZqYFNaHows=";
+      name = "gnupg-2.4-agent-Fix-a-memory-leak";
+    })
+
+    ### gpg.fail
+
+    # CVE-2025-68973: gpg.fail/memcpy (from upstream/master 115d138ba599328005c5321c0ef9f00355838ca9, cherry-picked to 2.4 branch)
+    ./gpg.fail-memcpy-CVE-2025-68973-2.4--gpg-Fix-possible-memory-corruption-in-the-armor-pars.patch
```

Comment:
Covered by https://dev.gnupg.org/rG4ecc5122f20e10c17172ed72f4fa46c784b5fb48 in 2.4.9